web/client/src/golden-layout/golden-layout-adapter.vue

```diff
@@ -105,26 +117,31 @@ const loadGLLayout = async (layoutConfig: LayoutConfig) => {
         | RowOrColumnItemConfig[]
         | StackItemConfig[]
         | ComponentItemConfig[];
+
         for (const itemConfig of content) {
-            if (itemConfig.type == 'component') {
-                index = addComponent(itemConfig.componentType as string);
-                if (typeof itemConfig.componentState === 'object' && itemConfig.componentState) {
-                    (itemConfig.componentState as any)['refId'] = index;
-                } else {
-                    itemConfig.componentState = { refId: index };
-                }
-            } else if (itemConfig.content.length > 0) {
+            if (itemConfig.type !== 'component') {
                 contents.push(itemConfig.content as
-                | RowOrColumnItemConfig[]
-                | StackItemConfig[]
-                | ComponentItemConfig[]);
+                    | RowOrColumnItemConfig[]
+                    | StackItemConfig[]
+                    | ComponentItemConfig[]);
+
+                continue;
```

Comment:
I think i wouldnt use ```continue;``` I think it makes the code harder to read.

Reply:
Do you have an alternative suggestion? What style would you prefer for the code? I used the early return/continue scheme 🤔

Reply:
Yeah i think in this case its the best solution. I missunderstood the method